docs/book/src/topics/addons.md

```diff
@@ -0,0 +1,158 @@
+# CNI
+
+By default, the CNI plugin is not installed for self-managed clusters, so you have to [install your own](https://cluster-api.sigs.k8s.io/user/quick-start.html#deploy-a-cni-solution).
+
+Some of the instructions below use [Helm](https://helm.sh) to install the addons. If you're not familiar with using Helm to manage Kubernetes applications as packages, there's lots of good [Helm documentation on the official website](https://helm.sh/docs/). You can install Helm by following the [official instructions](https://helm.sh/docs/intro/install/).
+
+## Calico
+
+To install [Calico](https://www.tigera.io/project-calico/) on a self-managed cluster using the office Calico Helm chart, run the commands corresponding to the cluster network configuration:
+
+### For IPv4 clusters
+
+```bash
+IPV4_CIDR_BLOCK=<cluster ipv4 pod cidr block> \
+helm repo add projectcalico https://projectcalico.docs.tigera.io/charts && \
+helm install calico projectcalico/tigera-operator -f https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/main/templates/addons/calico/values.yaml --set-string installation.calicoNetwork.ipPools[0].cidr="$IPV4_CIDR_BLOCK" --namespace tigera-operator --create-namespace
+```
+
+### For IPv6 clusters
+
+```bash
+IPV6_CIDR_BLOCK=<cluster ipv6 pod cidr block> \
+helm repo add projectcalico https://projectcalico.docs.tigera.io/charts && \
+helm install calico projectcalico/tigera-operator -f https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/main/templates/addons/calico-ipv6/values.yaml  --set-string installation.calicoNetwork.ipPools[0].cidr="$IPV6_CIDR_BLOCK" --namespace tigera-operator --create-namespace
+```
+
+### For IPv4/IPv6 dual-stack clusters
+
+```bash
+IPV4_CIDR_BLOCK=<cluster ipv4 pod cidr block> \
+IPV6_CIDR_BLOCK=<cluster ipv6 pod cidr block> \
+helm repo add projectcalico https://projectcalico.docs.tigera.io/charts && \
+helm install calico projectcalico/tigera-operator -f https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/main/templates/addons/calico-dual-stack/values.yaml --set-string installation.calicoNetwork.ipPools[0].cidr="$IPV4_CIDR_BLOCK",installation.calicoNetwork.ipPools[1].cidr="$IPV6_CIDR_BLOCK" --namespace tigera-operator --create-namespace
+```
+
+<aside class="note">
+
+<h1> Note </h1>
+
+For Windows nodes, you also need to copy the kubeadm-config configmap to the calico-system namespace so the calico-node-windows Daemonset can find it:
```

Comment:
@marosset I'm concerned that this will break upstream tests out there that rely on the CAPZ windows calico resources, or that it will confuse users. It works as a workaround for our tests but it's not a great long-term solution. It would be much better if we can figure out a way to solve https://github.com/kubernetes-sigs/sig-windows-tools/blob/master/hostprocess/calico/install/calico-install.ps1#L13 and not rely on the kubeadm-config configmap.

Reply:
I assume we're good with this @marosset, correct?